_adr/81/index.adoc

```diff
@@ -19,10 +19,243 @@ NOTE: https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used
+`medium`:: Ingress 60MB/s, egress 60MB/s, 2000 partitions, 1000 connections per instance, max 100 connections per second, 1MB max message size.
+`large`:: Ingress 90MB/s, egress 90MB/s, 3000 partitions, 1500 connections per instance, max 100 connections per second, 1MB max message size.
+
+In this example the functional attributes scales linearly, with the exception of connection rate and max message size.
+In general, linearity SHALL NOT be a requirement, even for those attributes which do scale up.
+For example, it would have been acceptable to define the `large` ingress as 100M/s, even though this is not 3× the `small` ingress, even when the other `large` functional parameters are 3× their `small` values.
+
+=== General principles
+
+* A fleet shard is required to provide only a subset of all series supported by the service globally. Indeed, a fleet share might possibly support only a single series one. For example a shard on AWS might support a different set of series to a shard on Azure, or the same series might have different sizes on those clouds.
+
+* The availability of a series might be limited to a particular cloud, region or combination of both.
+
+* It is a private concern of the shard exactly how each size in the series is provisioned. In principle it could use vertical, horizontal scaling and/or density > 1, in addition to applying quotas to limit capacity to the advertised limits. The fleet manager doesn't know.
+
+=== UI/Fleet manager interface
+
+This ADR doesn't mandate any particular presentation for how the scale options are presented in the UI.
+In particular while there is only one series defined, there is no requirement to display series in the UI at all.
+
+The fleet manager SHALL be the source of truth for the series and sizes available, and how those depend on cloud and region. 
+The fleet manager SHALL provide an endpoint which the MUST UI use to determine the available series and sizes.
+The UI MUST NOT make assumptions about linearity of scaling, instead it uses only the series and sizes provided by the endpoint.
+
+==== Series discovery
+
+Reusing the "Example" series defined above, let's illustrate how the Fleet Manager could communicate the available kinds of cluster to a UI.
+
+.Sequence diagram of UI/Fleet manager interaction
+image::ui-fleet_manager.png[UI/Fleet manager interaction]
+
+After the UI has discovered the available cloud providers and regions it would make another GET request `/api/kafkas_mgmt/v1/series/{cloudId}/{cloudRegion}` to discover the series available in this cloud and region.
+An example response JSON is shown below.
+
+[source,json]
+.Example series available to a UI from the Fleet Manager's `GET /api/kafkas_mgmt/v1/series/{cloudId}/{cloudRegion}` endpoint
+----
+"series": [ <1>
+    { "name": "Example", <2>
+      "id": 1, <3>
+      "description": "Multi-broker, multi-AZ", <4>
+      "sizes": [ <5>
```

Comment:
probably want a `defaultSize` field for each series.

Reply:
The need for a default arises from wanting to avoid a `v2` API. If we were happy to have v2 we'd just make it mandatory in the creation request. It feels a bit rubbish to need a default in the discovery response, just for that purpose. I feel like we could leave it out-of-band to the API, at least for now, since all existing clients of the API are only expecting to be able to create a "standard" instance anyway.